include/deal.II/base/parameter_handler.h

```diff
@@ -961,6 +961,11 @@ class ParameterHandler : public Subscriptor
      * values as a LaTeX file.
      */
     ShortLaTeX = LaTeX | Short,
+
+    /**
+     * Write out only parameters with changed values.
+     */
+    KeepChanged = 0x0200,
```

Comment:
Wouldn't `KeepOnlyChanged` make more sense here? Right now, to me this option reads like not keeping the changed parameters is the default.

Anyway, thanks for this feature, looking forward to it!